assets/src/editor/events/tickets/editor-ticket/price-calculator/ticket-price-calculator-form-data-map.js

```diff
@@ -5,6 +5,7 @@ import { isArray, isEmpty, uniq } from 'lodash';
 import { isModelEntityOfModel } from '@eventespresso/validators';
 
 import { TICKET_PRICE_CALCULATOR_FORM_INPUT_PREFIX } from './constants';
+import { shortenCuid } from '../../../../utils';
```

Comment:
`shortenCuid` seems like something that would be good to move into the  `@eventespresso/helpers` module.  I'd prefer it done in a separate pull so that any place using it can be updated in that pull and unit tests could also be added for it.

Reply:
sure, will add to EDTR project board